BuildScript.UnitTests/FilterTestConfigurationsTest.cs

```diff
@@ -237,12 +237,12 @@ public void SupportedBrowsers_FilterForUnsupportedBrowser_Error ()
       var testConfiguration = CreateTestConfiguration (browser: unsupportedBrowser);
       var testConfigurations = new[] { testConfiguration };
       var supportedBrowsers = new[] { "Chrome" };
-      var loggerMock = MockRepository.GenerateMock<ITaskLogger>();
-      loggerMock.Expect (_ => _.LogError ($"Cannot filter with unsupported {TestingConfigurationMetadata.Browser} '{unsupportedBrowser}'"));
-      var filter = CreateFilterTestConfigurations (supportedBrowsers: supportedBrowsers, logger: loggerMock);
+      var filter = CreateFilterTestConfigurations (supportedBrowsers: supportedBrowsers);
 
-      Assert.That (() => filter.GetFilteredTestConfigurations (testConfigurations), Throws.Exception);
-      loggerMock.VerifyAllExpectations();
+      Assert.That (
+          () => filter.GetFilteredTestConfigurations (testConfigurations),
+          Throws.InvalidOperationException.With.Message.EqualTo (
+              $"Cannot filter with unsupported {TestingConfigurationMetadata.Browser} '{unsupportedBrowser}'"));
```

Comment:
You could add a line break before `.With` instead of before the argument (multiple occurrences in this file).